setup.py

```diff
@@ -6,21 +6,21 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'Click',
-        'PyGithub',
-        'pyyaml',
-        'cerberus',
         'arrow',
+        'bs4',
+        'cerberus',
+        'Click',
+        'dedupe',
         'mergedeep',
-        'python-gitlab',
-        'tabulate',
```

Comment:
@LanderU this has been removed unfortunately. Let's please not submit PRs until we fix the existing issue.

Reply:
We need `tabulate` back again.

Reply:
Oh! It's below! my bad!